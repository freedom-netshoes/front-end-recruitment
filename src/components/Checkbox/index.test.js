import Checkbox from './';

describe('Checkbox', () => {
  it('should render correctly', () => {
    const wrapper = shallow(
      <Checkbox label="M" handleCheckboxChange={() => {}} />
    );

    expect(wrapper).toMatchSnapshot();
  });
});
