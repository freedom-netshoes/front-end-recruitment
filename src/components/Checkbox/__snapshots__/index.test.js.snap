// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Checkbox should render correctly 1`] = `
<div>
  <label>
    <input
      checked={false}
      onChange={[Function]}
      type="checkbox"
      value="M"
    />
    <span
      className="checkmark"
    >
      M
    </span>
  </label>
</div>
`;
